### ISSUE TITLE
ferries need a higher penalty in foot routing

### ISSUE THREAD

**Issue by utack** [mapsme/omim#4662]
Ferries currently seem to have a really low or no penalty in foot routing
This can lead to some odd results
Maybe you can add a default +10m waiting penalty to all ferries. It is not a great solution, but much closer to the real situation where you always need to wait at least some time

**Comment by burivuh** [comment]
You're right. We'll look into this with our major update of pedestrian routing in the beginning of 2017

**Comment by thany** [comment]
I didn't know it has ferry routing?..

Out of curiosity, where is it pulling timetables (and possibly pricetables) from?
